<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CANMessageWindowConfigDialog</class>
 <widget class="QDialog" name="CANMessageWindowConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAN message window configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/CAN.png</normaloff>:/Icons/CAN.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="WindowTitleGroupBox">
     <property name="title">
      <string>Window title</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="WindowTitleLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="AcceptanceMaskGroupBox">
     <property name="title">
      <string>Acceptance mask</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="AcceptanceMaskTableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="NewPushButton">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DeletePushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DisplayColumnsGroupBox">
     <property name="title">
      <string>Dispay columns</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="MessageCounterCheckBox">
        <property name="text">
         <string>Message counter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MessageTimeAbsoluteCheckBox">
        <property name="text">
         <string>Message time absolute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MessageTimeDifferenceCheckBox">
        <property name="text">
         <string>Message time difference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MessageTimeMinMaxDifferenceCheckBox">
        <property name="text">
         <string>Message time min/max difference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="RecorderGroupBox">
     <property name="title">
      <string>Recorder</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="RecorderOnCheckBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>on to file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RecorderFilenameLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="RecorderFilePushButton">
          <property name="text">
           <string>File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="RecorderTriggerCheckBox">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Trigger</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="RecorderTriggerEquationLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CANMessageWindowConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CANMessageWindowConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
